ui/src/assets/widgets/card.scss

```diff
@@ -41,4 +49,13 @@ $box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.15);
     border: unset;
     box-shadow: unset;
   }
```

Comment:
In a stack maybe avoid adding the drop shadow and just tint the background color grey? I think the drop shadow effect looks a little odd when in the stack.